### PR TITLE
add doc for tidb information schema tables

### PR DESCRIPTION
**Pull request by crazycs520**
### What is changed, added or deleted? 
add doc for tidb information schema tables

### CODE REVIEW

**Comment by crazycs520** [comment]
@yikeke PTAL

**Comment by yikeke** [comment]
> @yikeke PTAL

I‘ll review it later. You can ask for a technical review at the same time. Thanks!

**Comment by crazycs520** [comment]
@zimulala PTAL

File: dev/reference/system-databases/information-schema.md [hunk 1]
```diff
@@ -241,6 +274,45 @@ mysql> SELECT * FROM session_variables LIMIT 10;
 10 rows in set (0.00 sec)
 ```
 
+## SLOW_QUERY Table
+
+ `SLOW_QUERY` 提供了慢查询相关的一些信息。表里面的内容是通过解析 TiDB 慢日志文件的数据而来的，表中列名和慢日志中的字段名是一一对应了。更多操作可以参考 [慢查询日志文档](../../how-to/maintain/identify-slow-queries.md)。
```
Comment (tangenta) [inline dev/reference/system-databases/information-schema.md]:
```suggestion
 `SLOW_QUERY` 提供了慢查询相关的一些信息。表里面的内容是通过解析 TiDB 慢日志文件的数据而来的，表中列名和慢日志中的字段名是一一对应的关系。更多操作可以参考 [慢查询日志文档](../../how-to/maintain/identify-slow-queries.md)。
```

File: dev/reference/system-databases/information-schema.md [hunk 2]
```diff
@@ -200,6 +219,20 @@ POSITION_IN_UNIQUE_CONSTRAINT: NULL
 2 rows in set (0.00 sec)
 ```
 
+## PROCESSLIST Table
+
+ `PROCESSLIST` 和 `show processlist` 的功能一样，都是查看当前正在处理的请求。
```
Comment (winkyao) [inline dev/reference/system-databases/information-schema.md]: infoschema 里面的这个表会比 show processlist 多一个 mem 列

Comment (crazycs520) [reply dev/reference/system-databases/information-schema.md]: done.

**Pull request merged**